Dockerfile

```diff
@@ -6,16 +6,27 @@ FROM codait/max-base:v1.1.3
 
 WORKDIR /workspace
 
-RUN wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file} && \
-  tar -x -C assets/ -f assets/${model_file} -v && rm assets/${model_file}
+ARG use_pre_trained_model=true
+
+RUN if [ "$use_pre_trained_model" = "true" ] ; then\
+    # download pre-trained model artifacts from Cloud Object Storage
+    wget -nv --show-progress --progress=bar:force:noscroll ${model_bucket}/${model_file} --output-document=assets/${model_file} &&\
+            tar -x -C assets/ -f assets/${model_file} -v && rm assets/${model_file} ; fi
 
 COPY requirements.txt /workspace
 RUN pip install -r requirements.txt
 
 COPY . /workspace
 
-# check file integrity
-RUN md5sum -c md5sums.txt
+RUN if [ "$use_pre_trained_model" = "true" ] ; then \
+      # validate downloaded pre-trained model assets
+      md5sum -c md5sums.txt ; \
+    else \
+      # rename the directory that contains the custom-trained model artifacts
+      if [ -d "/workspace/custom_assets/" ] ; then \
+        mv /workspace/custom_assets/* /workspace/assets ; \
```

Comment:
Rather than using `mv`, which leads to an additional docker layer of the size of approximately the asset files (which can be really large). This [blog post](https://blog.dbi-services.com/docker-efficiently-building-images-for-large-software/) shows an interesting demonstration. How about use a symbolic link?

    rm -rf ./assets && ln -s ./custom_assets ./assets

Reply:
Makes sense.